docs/usage/configuration-options.md

```diff
@@ -1679,6 +1684,29 @@ Use this field to restrict rules to a particular language. e.g.
 }
 ```
 
+### matchCategories
+
+Use this field to restrict rules to a particular language or group. The categories can be found on the [manager documentation](./modules/manager.md). e.g.
+
+<!-- prettier-ignore -->
+!!! warning
+    Rules with `matchCategories` are only applied after extraction of a repository, therefore it can not be used to completely prevent analyzes of a repository.
+    Still it can be used to prevent or change the renovation of dependencies.
```

Comment:
```suggestion
    Still it can be used to prevent or change the updating of dependencies.
```

We should not create new verbs, as this confuses the reader.